src/main/java/com/faforever/client/game/CustomGamesController.java

```diff
@@ -191,13 +199,16 @@ protected void onDisplay(NavigateEvent navigateEvent) {
 
   private void updateFilteredItems() {
     preferencesService.storeInBackground();
+    GameFilterPrefs gameFilterPrefs = preferencesService.getPreferences().getGameFilterPrefs();
 
-    boolean showPasswordProtectedGames = showPasswordProtectedGamesCheckBox.isSelected();
-    boolean showModdedGames = showModdedGamesCheckBox.isSelected();
+    boolean showPasswordProtectedGames = gameFilterPrefs.isShowPasswordProtectedGames();
+    boolean showModdedGames = gameFilterPrefs.isShowModdedGames();
+    boolean showBlacklistedMaps = gameFilterPrefs.isShowBlacklistedMaps();
 
     filteredItems.setPredicate(OPEN_CUSTOM_GAMES_PREDICATE.and(gameInfoBean ->
-        (showPasswordProtectedGames || !gameInfoBean.isPasswordProtected())
-            && (showModdedGames || gameInfoBean.getSimMods().isEmpty())));
+            (showPasswordProtectedGames || !gameInfoBean.isPasswordProtected())
+            && (showModdedGames || gameInfoBean.getSimMods().isEmpty())
+            && (showBlacklistedMaps || !gameFilterPrefs.mapBlacklistProperty().contains(gameInfoBean.getMapFolderName()))));
```

Comment:
I think the folder name is only unique concerning the map version. But i think instead of blacklisting certain map version, people want to blacklist all versions of a map...

Reply:
Yeah unfortunately the game info message only contains the map folder name which includes the version number. I am going to change the blacklist button on the map card to do all versions of the map but am not sure if that is possible from the game tiles without querying the database

Reply:
or maybe ask the server devs to add it as a field